### DISCUSSION TITLE
UI Kit 2.1.1 licence key and Primefaces

### DISCUSSION THREAD

**Discussion by david-zed** [primefaces/community#140] [Figma UI Kit] [answered]
Hi, I have just purchased uiKit 2.1.1 but I seem to require Figma to use it but when I try to install Figma it wants a monthly fee, what is going on please

**Comment by onursenture** [comment, marked as answer]
Hello, 

Yes, as you can understand from its name, our Figma UI Kit requires Figma to use. However, you can use Figma for many tasks for free. For detailed information, you can check out this page on Figma's website: https://www.figma.com/pricing/
